modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/customer-portal/fragments/welcome-banner/index.html

```diff
@@ -20,8 +19,27 @@ <h2>${languageUtil.get(locale, "projects")}</h2>
 				${languageUtil.get(locale,
 				"select-a-project-to-see-your-subscriptions-activate-your-products-and-view-team-members")}
 			</div>
+
+			<div class="text-video">
+				<div class="text-description">Not sure how to start?</div>
+
+			<a href="https://help.liferay.com/hc/en-us/sections/8712397409421-Customer-Portal" target="_blank">
+					<button class="button">
+					<svg class="svg-style" width="16" height="16" padding-boto viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+						<mask height="14" id="mask0_13759_25805" maskUnits="userSpaceOnUse" style="mask-type: alpha" width="14" x="1" y="1">
+								<path clip-rule="evenodd" d="M1.3335 7.99967C1.3335 4.31967 4.32016 1.33301 8.00016 1.33301C11.6802 1.33301 14.6668 4.31967 14.6668 7.99967C14.6668 11.6797 11.6802 14.6663 8.00016 14.6663C4.32016 14.6663 1.3335 11.6797 1.3335 7.99967ZM7.20016 5.39967C6.98016 5.23301 6.66683 5.39301 6.66683 5.66634V10.333C6.66683 10.6063 6.98016 10.7663 7.20016 10.5997L10.3135 8.26634C10.4935 8.13301 10.4935 7.86634 10.3135 7.73301L7.20016 5.39967Z" fill="black" fill-rule="evenodd" />
+							</mask>
+
+							<g mask="url(#mask0_13759_25805)">
+								<rect fill="#0B5FFF" height="16" width="16" />
+							</g>
+						</svg>
+						Watch Onboarding Videos
```

Comment:
Add translation

${languageUtil.get(locale,	"watch-onboarding-videos")}

Reply:
@Aline160 could you move this <svg> </svg> to a file and use the <img /> tag?

See this example:

`<img class="customer-portal-welcome-img h-100 position-absolute w-100" src="[resources:banner_background_customer_portal.svg]" />`

You could use something like:

`<img src="[resources:my_new_svg_icon_here.svg]" />`

Reply:
Corrected